cfpb/design-manual
# Update project to use capital framework v1.x.x

Fixes Less compiling on this project and upgrades to use Capital Framework v1.x.x, removes dependency on grunt concat task. Based on https://github.com/cfpb/cfgov-refresh/pull/608

This is needed to continue work on #346 
## Removals
- some commented out stuff on form fields page
- concat task for cf-less which concatenated capital framework files
## Changes
- update bower dependency to capital framework ~1.0.0
- update main.less to point to capital-framework.less file
## Testing
- check out branch
- run npm install, grunt
## Review
- @Scotchester 
## Checklist
